aura/tenants/metrics_integration.py

```diff
@@ -0,0 +1,28 @@
+import click
+
+from aura.api_command import api_command
+from aura.api_repository import make_api_call
+from aura.config_repository import CLIConfig
+from aura.decorators import pass_config
+
+
+@api_command(
+    name="get-metrics-integration", help_text="Get metrics integration endpoint for a tenant"
```

Comment:
Agree, just following the convention for now. Let's wait for the suggestions if any.

Reply:
To be consistent the sucommand should be a verb. So I am fine with `get-metrics-integration`. The alternative would be to make `metrics-integration` a new resource and then have a command `aura metrics-integrations get`, but as I understand you are not planning to add any other endpoints like POST/DELETE ? Because then it would make sense to have `aura metrics-integration create` , `aura metrics-integration delete` etc. If not, then I would prefer having it as it is now, under tenants.